### ISSUE TITLE
S3DG_Pytorch.py dropout_keep_prob

### ISSUE THREAD

**Issue by aladdinpersson** [qijiezhao/s3d.pytorch#1]
I've tried to follow your implementation of S3D and the only problem I had when testing it on UCF101 is that the scores for each training example in a minibatch are all equal. I tried to figure out what the problem could be and I think it's problematic to have dropout_keep_prob = 1.

When reading the documentation for dropout3d: https://pytorch.org/docs/stable/nn.html#torch.nn.Dropout3d. It says that dropout_keep_prob is the probability of an element being zeroed.

I've started learning pytorch and deeplearning not too long ago, so I might be completely wrong, but I think dropout_keep_prob should be set to 0 as default instead of 1.

**Comment by Guocode** [comment]
I have tried your code on my training set,however i found that the convergence rate is very slow.I wonder whether you have the same problem.And do you have achieved the accuracy as that you describe.
